presto-cli/src/main/java/com/facebook/presto/cli/Help.java

```diff
@@ -28,8 +28,8 @@ public static String getHelpText()
                 "DESCRIBE <table>\n" +
                 "SHOW COLUMNS FROM <table>\n" +
                 "SHOW FUNCTIONS\n" +
-                "SHOW CATALOGS\n" +
-                "SHOW SCHEMAS\n" +
+                "SHOW CATALOGS [LIKE <pattern>]\n" +
+                "SHOW SCHEMAS [FROM <catalog>] [LIKE <pattern>]\n" +
                 "SHOW TABLES [LIKE <pattern>]\n" +
```

Comment:
I realized missing `[FROM <catalog>.<schema>]` here. But I assume it is better add it in a seperate PR.